democracylab/emails.py

```diff
@@ -205,12 +221,22 @@ def send_volunteer_application_email(volunteer_relation, is_reminder=False):
         project=project.project_name,
         role=role_text)
     email_template = HtmlEmailTemplate()\
-        .header("You Have a New Volunteer!")\
+        .subheader("Opportunity Information:")\
+        .text_line("Title: {role}".format(role=role_details.display_name))\
+        .text_line("Organization: {projectname}".format(projectname=project.project_name))\
+        .text_line("Date: {currentdate}".format(currentdate=datetime_to_string(timezone.now(), DateTimeFormats.DATE_LOCALIZED)))\
```

Comment:
The date format this outputs is mm/dd/yy, when the design request has Month DD, YYYY. Should that be `%B %d %Y` in DateTimeFormats?

Reply:
You can create another Enum with the appropriate format string (yours looks correct to me, just don't forget the comma!)